lua/wire/client/wire_expression2_editor.lua

```diff
@@ -1838,6 +1838,8 @@ function Editor:Close()
 	self:SetV(false)
 	self.chip = false
 
+	net.Start("IsEditing") net.WriteBool(false) net.SendToServer()
```

Comment:
why are you using a net message from the client here? Isn't it enough to use the console commands, which already do this?